doc-content/enterprise-only/project-data/preview-editor-latest-version-test-proc.adoc

```diff
@@ -0,0 +1,14 @@
+[id='preview-editor-latest-version-test-proc']
+= Switching between versions of a test scenario
+
+{CENTRAL} provides you the ability to switch between the various versions of a test scenario. Every time you save the scenario, a new version of the scenario is listed under *Latest Versions*. To use this feature you should save the test scenario file at least once.
```

Comment:
Missing comma:
"To use this feature you should save the test scenario file at least once." >>
"To use this feature, you must save the test scenario file at least once."